### PR TITLE
View Status location on Accession

### PR DESCRIPTION
**Pull request by constanzauanini**
- Added the status, the location and the notification sections at the top
- Also added the add quantity, age and viability links.
- Styles are not finished, I asked for more details on the Jira ticket. Also missing location icon.

Screen Shot 2022-09-01 at 12 20 47

### CODE REVIEW

File: src/components/accession2/Accession2View.tsx [hunk 1]
```diff
@@ -46,6 +60,46 @@ export default function Accession2View(): JSX.Element {
...
+        )}
+        {accession?.storageLocation && (
+          <Box display='flex' padding={(theme) => theme.spacing(0, 2)}>
+            <Icon name='info' className={classes.iconStyle} />
+            <Typography paddingLeft={1}>{accession.storageLocation}</Typography>
+          </Box>
+        )}
+        <Box display='flex' padding={(theme) => theme.spacing(0, 2)}>
+          <Icon name='notification' className={classes.iconStyle} />
+          <Typography paddingLeft={1}>Notification</Typography>
```
Comment (karthikbtf) [inline src/components/accession2/Accession2View.tsx:79]: strings

**Review by karthikbtf — APPROVED** [review]
good enough for intermediate work.. use strings for notification?

File: src/components/accession2/Accession2View.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/Accession2View.tsx:79]: The text here is an actual notification/reminder "End-drying Reminder in 25 days" for example. This is just to see how it looks

Comment (karthikbtf) [reply src/components/accession2/Accession2View.tsx:79]: Ok, got it.

**Pull request merged**